packages/rocketchat-ui/client/views/app/createChannel.html

```diff
@@ -14,10 +14,12 @@ <h1 class="create-channel__title">{{_ "Create_A_New_Channel"}}</h1>
 				<div class="create-channel__switches">
 					<div class="rc-switch">
 						<label class="rc-switch__label" tabindex="-1">
-							<input type="checkbox" class="rc-switch__input" name="type" value="p" checked>
-							<span class="rc-switch__button">
-							<span class="rc-switch__button-inside"></span>
-						</span>
+							{{#if canCreateBothTypes}}
+								<input type="checkbox" class="rc-switch__input" name="type" value="p" checked>
```

Comment:
instead of `{{#if canCreateBothTypes}}` use
` <input type="checkbox" class="rc-switch__input" name="type" value="p" checked={{testIfTypeEqualP}} disabled="{{cantCreateBothTypes}}">`